examples/ale_regression_boston.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>Would have been nice to set  <code>FEATURE</code>  to be a string and then do <code>feature_names.index(FEATURE)</code> to get the axis labels.  That way, the user can also redo your work for other features if they are interested and don't want to print the list and count the position of the index they care about...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p><code>feature_names</code>  is a numpy array though so it's a bit tricky, I've done this:</p><p> </p><pre class="ql-syntax" spellcheck="false">FEATURE = 'RM'
index = np.where(feature_names==FEATURE)[0][0]


fig, ax = plt.subplots()
ax.scatter(X_train[:, index], lr.predict(X_train));


ax.set_xlabel(FEATURE);
ax.set_ylabel('Value in $1000\'s');
</pre>